### DISCUSSION TITLE
Table - useTable | Lack of pointer events in table causing issues for other components.

### DISCUSSION THREAD

**Discussion by Zurdge** [adobe/react-spectrum#5128] [General]
We've been using  `useTable` to create our Table component in our UI library.
when the table has selectable checkboxes we're not getting pointer-events passing through to our other nested components which are inside some of the Table cells.
Has anyone else had this kind of issue? Are there a way to allow events to propagate through?

**Comment by reidbarber** [comment]
For enabling text selection in a cell, see this section of the FAQ: [How can I select text in a TableView cell?](https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#how-can-i-select-text-in-a-tableview-cell)

I'm not seeing that behavior you're describing in this sandbox example, however: https://codesandbox.io/s/usetable-with-cell-actions-example-fkszhz?file=/src/App.js

Can you try reproducing this issue in a sandbox? (feel free to fork that sandbox if needed).
